
  <include file="$(find autorally_core)/launch/hardware.machine" />
  <node name="gpsRover" pkg="autorally_core" type="gpsReachInterface" output="screen" machine="autorally-master">

    <!-- mode can be either "base" or "rover" -->
    <param name="mode" value="rover" />
    <param name="accuracyRTK" value="0.02" />
    <param name="accuracyWAAS" value="0.6" />
    <param name="accuracyAutonomous" value="2.5" />
    <param name="utcSource" value="GPZDA" />
    <param name="statusPositionSource" value="GPGNS" /> <!-- GPGNS | GPGGA -->
    <param name="showGsv" value="false" />
    
    <!--configure settings for primary port -->
    <param name="primaryPort/portPath" value="/dev/arGPSroverPortA" />
    <param name="primaryPort/serialBaud" value="115200" />
    <param name="primaryPort/serialDataBits" value="8" />
    <param name="primaryPort/serialParity" value="none" />
    <param name="primaryPort/serialStopBits" value="1" />
    <param name="primaryPort/serialHardwareFlow" value="false" />
    <param name="primaryPort/serialSoftwareFlow" value="false" />

  </node>

</launch>
